### PR TITLE
Search

### PR DESCRIPTION
**Pull request by mohammed-sas**
[preview](https://deploy-preview-9--mhd-racquets-store.netlify.app/)

1.added search functionality
2. search by 
    a.category : racquet , shuttlecock , string and shoe
   b. brand : yonex

PS
1.clicking on searched item wont redirect to product page , i will impelment in the upcoming PR
2. responsiveness and UI improvement would done at the end
3. feedbacks are most welcomed

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -3,13 +3,31 @@ import logo from "../../assets/ecomm-logo.png";
...
   const { currentUser, signout } = useAuth();
+  const [searchResult, setSearchResult] = useState([]);
+  const searchHandler = (e) => {
+    let inputVal = e.target.value.toLowerCase().trim();
+    let inputLen =inputVal.length;
+    let result = state.data.filter(
+      (item) =>
+        item.title.toLowerCase().substring(0,inputLen).includes(inputVal) ||
+        item.categoryName.toLowerCase().substring(0,inputLen).includes(inputVal)||
+        item.brand.toLowerCase().substring(0,inputLen).includes(inputVal)
```
Comment (AtulPant2704) [inline src/components/Navbar/Navbar.jsx:25]: Loved the way of writing code, you made it really easy to understand.

**Review by AtulPant2704 — APPROVED** [review]
Was aware of how to search by an item, but you combined search by brand and category really well, got to learn something new from it.

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (mohammed-sas) [reply src/components/Navbar/Navbar.jsx:25]: thank you for the appreciation atul

**Review by starver20 — APPROVED** [review]
Nicely implemented, good to merge.

**Pull request merged**